apps/src/templates/instructions/codeReview/ReviewNavigator.jsx

```diff
@@ -19,6 +21,25 @@ class ReviewNavigator extends Component {
     loadInProgress: false
   };
 
+  onSelectPeer = peer => {
+    if (!peer.id) {
+      return;
+    }
+
+    navigateToHref(
```

Comment:
Somewhere, I think it might be nice to have a comment explaining why we have to navigate to a new page when a peer is selected since this is a bit unusual.

Also, is there any risk of lost work when the student clicks on a peer and navigates away?  Do we save or warn the user before navigating away?

Reply:
From a quick glance at the current functionality, if you try to navigate away after editing the project there is a warning. If you start writing a comment and navigate away there is no warning.